_integration-schemas/hubspot/companies.md

```diff
@@ -1,18 +1,16 @@
 ---
 tap: "hubspot"
-version: "1.0"
+version: "2.0"
 
 name: "companies"
 doc-link: http://developers.hubspot.com/docs/methods/companies/get_company
 singer-schema: https://github.com/singer-io/tap-hubspot/blob/master/tap_hubspot/schemas/companies.json
 description: |
   The `companies` table contains info about the companies your HubSpot contacts belong to.
 
-  **Note:** when this table is synced, the [`hubspot_contacts_by_company`](#hubspot_contacts_by_company) table will also be automatically synced and created in your data warehouse.
+  **Note:** When this table is set to replicate, the [`contacts_by_company`](#contacts_by_company) table will be automatically created in your data warehouse.
```

Comment:
This is actually not true. We have table selection here so you can choose whether or not to sync this table.

Reply:
Oooh, this is great! Before this wasn't selectable in the UI. I'll make this change in just a sec, but then these docs are good to go.